### ISSUE TITLE
Make enemy colors match room colors in Tower tileset

### ISSUE THREAD

**Issue by R3Ked** [v6cord/VVVVVV-CE#8]
Ok, so since there are more than 6 colors in the Tower tileset, enemy colors should match, right? This is not the case, however. Enemy colors are stuck to the original 6 colors, as clearly shown in this image.
![colorbug](https://user-images.githubusercontent.com/20342336/78516034-03603a00-776d-11ea-8b2f-f04e99918097.PNG)
As far as I know, this only happens in the Tower tileset, since there are only 6 colors (7 in Warp Zone's case) for things to be in.

**Comment by InfoTeddy** [comment]
We would have to add 30 colors for each gradation of Tower tileset color, which could get a bit unwieldy.

Colors aren't as simple as having an R, G, and B. There's a color function that makes colors glow and animate. Colors aren't static or anything.

**Comment by R3Ked** [comment]
I'm not a coder or anything, but don't you do this sort of thing already? There only exists one color of each sprite, and it just gets recolored for each color option. Can't you do that with the other colors that the tower cycles through?
Although, that would be a lot in the code.

**Comment by InfoTeddy** [comment]
No, that's not how the color system in VVVVVV works. There's not like a "master color" that determines the set of colors for each sprite. Each sprite color has to be manually set for each color of a tileset.

The point remains that there's no already-existing sprite color to fill the colors in-between the "stable" colors of the Tower Hallway tileset.
